lab_1/main.py

```diff
@@ -62,10 +119,29 @@ def compare_profiles(unknown_profile: dict, profile_to_compare: dict, top_n: int
     :param top_n: a number of the most common words
     :return: the distance
     """
-    pass
+    if not isinstance(unknown_profile, dict) or not isinstance(profile_to_compare, dict):
+        return None
+    if not isinstance(top_n, int):
+        return None
+
+    freq_dict_unk = unknown_profile['freq']
+    freq_dict = profile_to_compare['freq']
+
+    top_n_words_unk = get_top_n_words(freq_dict_unk, top_n)
```

Comment:
no need to store ` freq_dict_unk` and  `freq_dict`. You can use a dictionary (`unknown_profile['freq']`) just here

Reply:
done